crates/octez/src/node.rs

```diff
@@ -39,7 +39,7 @@ impl OctezNode {
             self.octez_node_dir.to_str().expect("Invalid path"),
             "--net-addr",
             http_endpoint,
-            "--rpc-addr",
+            "--local-rpc-addr", // TODO: @alanmarkoTrilitech --local-rpc-addr is not present in future versions of octez-node
```

Comment:
The purpose of adding --singleprocess is to make it so that octez-validator is run together with octez-node, thus killing it is easier. This only works when --local-rpc-addr is provided, as we have decided to work with a specific octez commit instead of master. In later versions --rpc-addr should be used as it was merged into a single flag. I don't think it would be productive to make the switch now, we can do it though right before releasing the crate for dog fooding.